### PR TITLE
feat: team page layout added

### LINKED ISSUE #214: [Feature] : New team page
### Description

Team page for DevsInTech Members

### Screenshots

image

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by AbhiPatel10**
## Fixes Issue

close #214 

## Changes proposed

New team page added

## Screenshots

Screenshot from 2023-06-03 17-14-23

### CODE REVIEW

File: src/components/TeamPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+import Link from "next/link";
+import { FaGithub, FaLinkedin, FaTwitter } from "react-icons/fa";
+import TeamData from "../content/team";
+import Header from "../header";
+
+const TeamPage = () => {
+  console.log("team ", TeamData[0]);
```
Comment (prnvtripathi) [inline src/components/TeamPage/index.jsx:8]: I think we can remove this console log from here.

**Review by prnvtripathi — COMMENTED** [review]
Looks good just one line, which i have requested to change

File: src/components/TeamPage/index.jsx (hunk 1 above)
Comment (AbhiPatel10) [reply src/components/TeamPage/index.jsx:8]: Yes, In Hurry i forgot to remove, will do that

Comment (AbhiPatel10) [reply src/components/TeamPage/index.jsx]: Done

**Review by prnvtripathi — APPROVED** [review]
Looks good to merge ☺️

**Pull request merged**